src/components/ModalBox/BoxTitle.js

```diff
@@ -18,55 +18,50 @@
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 import * as React from 'react';
-import styled, { withTheme } from 'styled-components/native';
-
-import type { Theme } from 'models/Theme';
+import styled from 'styled-components/native';
 
 // components
 import { MediumText } from 'components/Typography';
 import IconButton from 'components/IconButton';
 
 // utils
 import { fontSizes, spacing } from 'utils/variables';
-import { getThemeColors } from 'utils/themes';
 
 type Props = {
   title: string,
   onPressClose: () => void,
-  theme: Theme,
 };
 
 const Wrapper = styled.View`
+  align-items: flex-start;
   display: flex;
   width: 100%;
+  padding: 0;
   flex-direction: row;
 `;
 
 const Title = styled(MediumText)`
-  margin: ${spacing.large}px;
   flex: 1;
+  margin: ${spacing.rhythm}px;
+  margin-bottom: ${spacing.small};
   font-size: ${fontSizes.big}px;
 `;
 
-const CloseIcon = styled(IconButton)`
-  padding: ${spacing.large}px;
-  width: 58px;
+const CloseIcon = styled(IconButton).attrs({
+  icon: 'close',
+  fontSize: fontSizes.regular,
+  secondary: true,
+})`
+  margin: ${spacing.layoutSides}px;
```

Comment:
Can it be 'padding' instead of 'margin'? It would increase icon's clickable area :)

Reply:
Yup, that's better!